tests/manage/z_cluster/nodes/test_rolling_shutdown_and_recovery.py

```diff
@@ -0,0 +1,79 @@
+import logging
+import time
+import pytest
+
+
+from ocs_ci.framework.testlib import (
+    tier4c,
+    ignore_leftovers,
+    ManageTest,
+    bugzilla,
+    skipif_external_mode,
+    skipif_ibm_cloud,
+)
+from ocs_ci.ocs.node import get_ocs_nodes
+from ocs_ci.ocs.resources.pod import wait_for_pods_to_be_running
+from ocs_ci.helpers.sanity_helpers import Sanity
+
+
+log = logging.getLogger(__name__)
+
+
+@tier4c
+@pytest.mark.polarion_id("OCS-2633")
+@bugzilla("1895819")
+@skipif_ibm_cloud
+@skipif_external_mode
+@ignore_leftovers
+class TestRollingWorkerNodeShutdownAndRecovery(ManageTest):
```

Comment:
the test is quite similar to tests.manage.cluster.nodes.test_nodes_restart.TestNodesRestart.test_rolling_nodes_restart

Do we need both?

Reply:
@ebenahar CU hit the SIGSEG on a mon node during node shutdown. Neha ojha from Dev who fixed the issue suggested us to have test cases for both shutdown and reboot case and hence I added this new test for rolling node shutdown.